css/style.css

```diff
@@ -1452,4 +1452,26 @@ button.connect-secondary {
     position: absolute;
     right: 10px;
     cursor: pointer;
+}
+
+.screen-reader-only {
```

Comment:
Would it make more sense to define these together as `.screen-reader-only, .visually-hidden-for-screen-reader`, since they seem to have identical properties?

Reply:
Great catch - yes, cleaned this up. Thanks.